docs/core/compatibility/core-libraries/5.0/environment-osversion-returns-correct-version.md

```diff
@@ -9,14 +9,16 @@ ms.date: 11/01/2020
 
 ## Change description
 
-In previous .NET versions, <xref:System.Environment.OSVersion?displayProperty=nameWithType> returns an OS version that may be incorrect when an application runs under Windows compatibility mode. For more information, see [GetVersionExA function remarks](/windows/win32/api/sysinfoapi/nf-sysinfoapi-getversionexa#remarks).
+In previous .NET versions, <xref:System.Environment.OSVersion?displayProperty=nameWithType> returns an OS version that may be incorrect when an application runs under Windows compatibility mode. For more information, see [GetVersionExA function remarks](/windows/win32/api/sysinfoapi/nf-sysinfoapi-getversionexa#remarks). On macOS, it returns the underlying Darwin kernel version.
 
-Starting in .NET 5.0, <xref:System.Environment.OSVersion?displayProperty=nameWithType> returns the actual version of the operating system.
+Starting in .NET 5.0, <xref:System.Environment.OSVersion?displayProperty=nameWithType> returns the actual version of the operating system for Windows and macOS.
 
 ## Reason for change
 
 Users of this property expect it to return the actual version of the operating system. Most .NET apps don't specify their supported version in their application manifest, and thus get the default supported version from the dotnet host. As a result, the compatibility shim is rarely meaningful for the app that's running. When Windows releases a new version and an older dotnet host is still in use, these apps may get an incorrect OS version. Returning the actual version is more inline with developers' expectations of this API.
 
+With the introduction of <xref:System.OperatingSystem.IsWindowsVersionAtLeast?displayProperty=nameWithType>, <xref:System.OperatingSystem.IsMacOSVersionAtLeast?displayProperty=nameWithType>, and <xref:System.Runtime.Versioning.SupportedOSPlatformAttribute?displayProperty=nameWithType> in .NET 5.0, <xref:System.Environment.OSVersion?displayProperty=nameWithType> was changed to be consistent for Windows and macOS.
```

Comment:
@gewarren -

The links to `<xref:System.OperatingSystem.IsWindowsVersionAtLeast?displayProperty=nameWithType>` and `<xref:System.OperatingSystem.IsMacOSVersionAtLeast?displayProperty=nameWithType>` don't seem to be working. Do I need to specify the parameter types?

I see other places using `%2A` at the end of a method xref. Is that what I should use?

Reply:
Yes, for methods, you either need to specify an exact overload or add %2A at the end, which links to the "overloads" section, so it applies to all overloads.

BTW, if you use VS Code, you can install the [Docs Markdown extension](https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-markdown), which makes adding xref links a no-brainer:

![image](https://user-images.githubusercontent.com/24882762/101666410-0948a200-3a03-11eb-87b8-f86b12aa89cd.png)

Reply:
Thanks for the help, will update.

> you can install the Docs Markdown extension

Oh nice! I'll give that a try - I've been using the GitHub UI to edit 😕